### ISSUE TITLE
Hyper-V Support

### ISSUE THREAD

**Issue by Mesagoppinmypants** [lordkator/ZonamaDev#153]
Does this support Hyper-V? If it doesn't then it would be probably be a good idea to add it.

**Comment by karlbunch** [comment]
I don't have a way to test would be happy to work with whomever wants to test and submit changes..

**Comment by Mesagoppinmypants** [comment]
If you can get it on the vhd file, I know someone who can help you test it.

**Comment by Marott1** [comment]
hi there i can help out to try and get it to run on hyper-v i do have a win server 2012R2 running where we can test it on

**Comment by phoenix79** [comment]
I'd be more than happy to try to test this on Hyper-V

**Comment by madpeanut** [comment]
was this ever tested?

**Comment by madpeanut** [comment]
actually, it looks like I got it figured out.  I'm still setting up but now that I have a hyper-v image I might be able to uninstall the other software.  I got some errors with the VM loading and the Linux server looks like it wants some virtual box references and there was a login service error(?) but I'll see if I can get it to load and login remotely.

**Comment by karlbunch** [comment]
Are you using vagrant for Hyper-V or building the base image by hand?

**Comment by madpeanut** [comment]
I did the setup as mentioned using virtualbox and then I converted the vmdk to a vhdk using Hyper-V's import capabilities.  I got it to mostly work but its missing TRE files and some other things on the Linux server and just haven't have time to put into it any further.  there are some errors on loading the Linux server but it seems to work okay as its just looking for core components for virtualbox and not designed with Hyper-V in mind.
